routing_model/Cargo.toml

```diff
@@ -1,11 +1,15 @@
 [package]
+authors = ["MaidSafe Developers <dev@maidsafe.net>"]
+description = "A secured storage DHT"
+homepage = "https://maidsafe.net"
+license = "GPL-3.0"
 name = "routing_model"
+repository = "https://github.com/maidsafe/routing"
 version = "0.1.0"
-authors = ["Jean-Philippe DUFRAIGNE <jean.dufraigne@maidsafe.net>"]
 edition = "2018"
 
-[dependencies]
-lazy_static = "~1.0.1"
-unwrap = "~1.2.1"
-itertools = "~0.8.0"
+[dev-dependencies]
+lazy_static = "1.0.1"
```

Comment:
i thought we were supposed to have the `~`

Reply:
Yeah, I thought so too, but Routing seems to be doing its own thing; different from our other crates.

Reply:
I thought "we" said that routing needing fixing at some point.

Reply:
did you meant to keep that?

Reply:
I was waiting for an answer in Slack.

Reply:
Yup - I'll put the tildes back in.  Routing will get a similar update sometime too.